test/goldens/end2end/webdriver-3.0.0.json_report.md

```diff
@@ -25,29 +25,6 @@ See [package layout](https://dart.dev/tools/pub/package-layout#examples) guideli
 
 ### [x] 0/20 points: Supports 0 of 6 possible platforms (iOS, Android, Web, Windows, MacOS, Linux)
 
-SDK issues found:
```

Comment:
This seems to be off to be removed.

Reply:
We decided on a more lenient strategy for assigning sdk's. So in broad terms you get "dart" if you don't import 'dart:ui' and you get flutter if you don't import 'dart:mirrors'.